hdf5-sys/src/h5es.rs

```diff
@@ -0,0 +1,87 @@
+//! Event set module
+use crate::internal_prelude::*;
+
+#[repr(C)]
+#[derive(Debug)]
```

Comment:
All the structs should probably be `Copy / Clone` (even if they contain pointers) because they are such in C